app/client/ui/HomeImports.ts

```diff
@@ -39,9 +40,9 @@ export async function fileImport(
     const timezone = await guessTimezone();
 
     if (workspaceId === "unsaved") {
-      function onUploadProgress(ev: ProgressEvent) {
-        if (ev.lengthComputable) {
-          progress.setUploadProgress(ev.loaded / ev.total * 100);   // percentage complete
+      function onUploadProgress(ev: AxiosProgressEvent) {
+        if (ev.event.lengthComputable) {
```

Comment:
FWIW, I achieved to hit a breakpoint here. The `workspaceId` equals to `"unsaved"` when the user is not logged in but is allowed to upload documents (temporary).

The values are the one that are expected.